test/git/scenarios.json

```diff
@@ -43,5 +43,86 @@
                 "ppa": "false"
             }
         }
+    },
+    "install_git_from_src_centos-7": {
+        "image": "centos:centos7",
+        "features": {
+            "git": {
+                "version": "latest",
+                "ppa": "false"
```

Comment:
Can we also add similar test scenarios with `ppa: true`? Thank you! 

This way we can also validate the code changes against 👇 

(TLDR: Make sure `git` [can be installed fine with the package managers](https://github.com/devcontainers/features/blob/3e96db3a6a28667bdc012ebecc3c798ddad6af99/src/git/install.sh#L172) With `ppa:false` `git` gets installed from source.)


https://github.com/devcontainers/features/blob/3e96db3a6a28667bdc012ebecc3c798ddad6af99/src/git/install.sh#L166-L176

Reply:
From https://launchpad.net/ubuntu/+ppas, Personal Package Archives (PPA) allow you to upload Ubuntu source packages to be built and published as an apt repository by Launchpad. You can find out more about PPAs and how to use them in our [help page](https://help.launchpad.net/Packaging/PPA).

Unfortunately PPAs are not for RedHat based systems so I cannot add these tests.

I'm not sure what the equivalent approach would be for RedHat systems: to my knowledge are no standard community package archives broadly accepted by corporations that use RedHat derived systems, and even those that exist tend not to keep broad ranges of versions of a package. 

I could add tests where `ppa=false` and `version=system`, which will use the system's package manager to pull the latest version available via the standard vendor (RedHat, Alma, Microsoft, Fedora, Rocky) repositories.

Would that be acceptable?

Reply:
Alternately, I could either set `ppa=true` in the scenarios or remove the `ppa=false` from the scenarios, and it still won't affect the build or test because PPAs only apply to Ubuntu.

Reply:
> Unfortunately PPAs are not for RedHat based systems so I cannot add these tests.

Thanks for pointing that out, really helpful. 

Can we update the `ppa` option description to explicitly specify that it is only available for `Ubuntu`?  This way we can avoid such similar (future) confusions. (eg. `Install from PPA if available (only supported for Ubuntu distributions`)

https://github.com/devcontainers/features/blob/3e96db3a6a28667bdc012ebecc3c798ddad6af99/src/git/devcontainer-feature.json#L20

> I could add tests where ppa=false and version=system, which will use the system's package manager to pull the latest version available via the standard vendor (RedHat, Alma, Microsoft, Fedora, Rocky) repositories.
> Would that be acceptable?

Yep, that's a good enough test case and definitely makes sense. Thank you!

Reply:
> Alternately, I could either set ppa=true in the scenarios or remove the ppa=false from the scenarios, and it still won't affect the build or test because PPAs only apply to Ubuntu: see lines 165 & 166 of install.sh.

Yep, so ideally we need two test scenarios for each distro with `version=latest` and `version=system` (which is default ; so can be omitted too)

Reply:
RedHat, Alma, Microsoft, Fedora, Rocky for building from source (`ppa=false` and `version=latest`) as well as installing via system packages (`ppa=true` and `version=system`) are in place and should be passing as of commit 0bd3add1.